### PR TITLE
Feat: add auth dev

### PR DESCRIPTION
**Pull request by wangbow**
Signed-off-by: wb-wb665667 <wb-wb665667@alibaba-inc.com>

### Description of your changes

Fixes  auth implements

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary.
- [x] Run `yarn lint` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: src/api/request.ts [hunk 1]
```diff
@@ -28,18 +33,46 @@ axiosInstance.interceptors.response.use(
...
+  async (error: any) => {
+    const { data, status } = error?.response || error;
+    const refreshToken = localStorage.getItem('refreshToken');
+    if (refreshToken && data.BusinessCode === '12002') {
+      try {
+        const res: any = await axios({
+          url: `${baseURL}/${authenticationRefreshToken}`,
+          method: 'GET',
+          headers: {
+            Authorization: `Bearer ${refreshToken}`,
```
Comment (barnettZQG) [inline src/api/request.ts:45]:
```suggestion
            ParseToken: refreshToken,
```

File: src/api/request.ts [hunk 2]
```diff
@@ -28,18 +33,46 @@ axiosInstance.interceptors.response.use(
...
+          },
+        });
+        if (res && res.accessToken) {
+          localStorage.setItem('token', res.accessToken);
+          localStorage.setItem('refreshToken', res.refreshToken);
+        }
```
Comment (barnettZQG) [inline src/api/request.ts:51]: If can not refresh token, should redirect to the login page.

File: src/pages/Login/index.tsx [hunk 3]
```diff
@@ -1,11 +1,222 @@
...
+        }
+      })
+      .catch();
+  };
+  onUpdateSystemInfo = (type: string) => {
+    updateSystemInfo({ enableCollection: true, loginType: type })
+      .then(() => {})
+      .catch();
+  };
+  getCode = () => {
```
Comment (barnettZQG) [inline src/pages/Login/index.tsx:92]: unuseful code?

Comment (wangbow) [reply src/pages/Login/index.tsx]: done

File: src/api/request.ts (hunk 2 above)
Comment (wangbow) [reply src/api/request.ts:57]: done

File: src/api/request.ts (hunk 1 above)
Comment (wangbow) [reply src/api/request.ts]: done

**Review by barnettZQG — APPROVED** [review]
LGTM

**Pull request merged**